debian/postinst

```diff
@@ -20,10 +20,15 @@ fi
 update-initramfs -u -k all
 
 if [ -f /etc/default/grub ]; then
-    sed -i '/\(^\|#\)GRUB_CMDLINE_LINUX=/c\GRUB_CMDLINE_LINUX="boot=startos"' /etc/default/grub
+    sed -i '/\(^\|#\)GRUB_CMDLINE_LINUX=/c\GRUB_CMDLINE_LINUX="boot=startos console=ttyS0,115200n8"' /etc/default/grub
     sed -i '/\(^\|#\)GRUB_DISTRIBUTOR=/c\GRUB_DISTRIBUTOR="StartOS v$(cat /usr/lib/startos/VERSION.txt)"' /etc/default/grub
+    sed -i '/\(^\|#\)GRUB_TERMINAL=/c\GRUB_TERMINAL="serial"\nGRUB_SERIAL_COMMAND="serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1"' /etc/default/grub
 fi
 
+# set local and remote login prompt
+echo "StartOS v$(cat /usr/lib/startos/VERSION.txt) [\m] on \n.local (\l)" > /etc/issue
```

Comment:
What are \m and \l?

Reply:
\m and \l are placeholders used in the /etc/issue file to dynamically display system information in the login prompt.
	•	\m: Displays the architecture of the machine (e.g., x86_64 or armv7l).
	•	\l: Displays the name of the device or terminal (e.g., tty1 for a local console or a specific PTY for remote logins).

Here, take a look at this example:
```
StartOS v0.3.6-alpha.8 [x86_64] on pregame-molasses.local (ttyS0)
```

Reply:
Ah, so `\n` is the hostname, not a newline then